src/Pos/ForwardClient/Types.hs

```diff
@@ -84,6 +84,7 @@ instance ToJSON CrTicket where
   toJSON CrTicket {..} =
         object [ "ticket" .=
             object [ "type"         .= ("custom report" :: Text)
+                    , "status"      .= ("open" :: Text)
```

Comment:
should be "new", if I understand DEVOPS-1143 correctly

Reply:
Thanks a lot, Jean-Baptiste!